content/docs/handling-events.md

```diff
@@ -1,43 +1,43 @@
 ---
 id: handling-events
-title: Handling Events
+title: Penanganan Events
 permalink: docs/handling-events.html
 prev: state-and-lifecycle.html
 next: conditional-rendering.html
 redirect_from:
   - "docs/events-ko-KR.html"
 ---
 
-Handling events with React elements is very similar to handling events on DOM elements. There are some syntactic differences:
+Menangani sebuah *events* dengan elemen React sangat mirip seperti menangani sebuah *events* pada elemen DOM. Ada beberapa perbedaan sintaks:
 
-* React events are named using camelCase, rather than lowercase.
-* With JSX you pass a function as the event handler, rather than a string.
+* *Events* pada React biasanya dituliskan dalam bentuk camelCase, dan bukan lowercase.
+* Dengan JSX Anda dapat oper sebuah *function* sebagai *event handler*, dan bukan sebuah *string*.
```

Comment:
```suggestion
* Dengan JSX Anda dapat mengoper *function* sebagai *event handler*, bukan sebagai *string*.
```

Reply:
Wasn't sure about _oper_, but [KBBI](https://kbbi.kemdikbud.go.id/entri/oper) does have it! _mengoper_ sounds good to me.

Reply:
okay, it's sounds good to me too.